config/manufacturer_specific.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ManufacturerSpecificData Revision="151" xmlns="https://github.com/OpenZWave/open-zwave">
+<ManufacturerSpecificData Revision="152" xmlns="https://github.com/OpenZWave/open-zwave">
```

Comment:
I'm not sure why the test fails at this stage. I have bumped the manufacturer_specific.xm revision